src/pyramid_googleauth/security.py

```diff
@@ -0,0 +1,21 @@
+from pyramid.authentication import SessionAuthenticationHelper
+
+
+class GoogleSecurityPolicy:
+    def __init__(self):
+        self._session_authentication_helper = SessionAuthenticationHelper()
```

Comment:
I think we may want to customize this so that it doesn't conflict with any session storage potentially being used by the app:

```suggestion
        self._session_authentication_helper = SessionAuthenticationHelper(prefix="googleauth".)
```

This also means that the app has to have a Pyramid session registered in order for this extension to work, we should add that to the README.

Reply:
> I think we may want to customize this so that it doesn't conflict with any session storage potentially being used by the app:
> 
> This also means that the app has to have a Pyramid session registered in order for this extension to work, we should add that to the README.

Changed this to come from a method and added the prefix in the default implementation.